### PR TITLE
Timeout on large table diffs

### PR DESCRIPTION
**Pull request by GavinMendelGleason**

### CODE REVIEW

File: src/core/api/api_error.pl [hunk 1]
```diff
@@ -1087,6 +1087,13 @@
                               'api:key' : Key},
              'api:message' : Msg
             }.
+api_error_jsonld_(diff,error(table_diff_timeout,_), JSON) :-
+    format(string(Msg), "The table diff you requested is too complex for our little brains. Perhaps try signing up for a dedicated table diff account?", []),
```
Comment (matko) [inline src/core/api/api_error.pl:1091]: lol. not sure if I'd phrase it that way but it'll do.

File: src/core/api/api_error.pl [hunk 2]
```diff
@@ -1087,6 +1087,13 @@
...
+    JSON = _{'@type' : 'api:DiffErrorResponse',
+             'api:status' : 'api:i_am_a_teapot',
```
Comment (matko) [inline src/core/api/api_error.pl:1093]: :D But really, we shouldn't be using this, as terminusdb is in fact not a teapot. Considering this message is trying to push people onto a paid tier, maybe 402 payment required? Apparently there's precedent - wikipedia says google uses it when people exceed limits on their APIs.

**Review by matko — CHANGES_REQUESTED** [review]
As much as I love teapots, I really do want another status code here. Please.

File: src/core/api/api_error.pl (hunk 2 above)
Comment (GavinMendelGleason) [reply src/core/api/api_error.pl]: payment required it is!

**Review by matko — APPROVED** [review]

**Pull request merged**